apps/api/src/app/testing/usecases/seed-data/seed-data.usecase.ts

```diff
@@ -10,15 +10,17 @@ export class SeedData {
   constructor(private authService: AuthService, private userRegister: UserRegister) {}
 
   async execute(command: SeedDataCommand) {
-    const { user } = await this.userRegister.execute(
-      UserRegisterCommand.create({
-        email: 'test-user-1@example.com',
-        firstName: faker.name.firstName(),
-        lastName: faker.name.lastName(),
-        password: '123qwe!@#',
-        organizationName: 'Test Organization',
-      })
-    );
+    const data = {
+      email: 'test-user-1@example.com',
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      password: '123qwe!@#',
+      organizationName: 'Test Organization',
+    };
+
+    const userRegisterCommand = UserRegisterCommand.create(data);
+
+    const { user } = await this.userRegister.execute(userRegisterCommand);
```

Comment:
While debugging to find some problems I splitted the code like this for easier debugging. No functionality changes.

Reply:
This is perfect 🚀